src/pattern-library/app.js

```diff
@@ -0,0 +1,21 @@
+// Entry point for extending the pattern library
```

Comment:
This is...sorta new but mostly just adapted. Wraps the app startup for use by other projects, eventually.